boards/arm/efr32_radio/doc/index.rst

```diff
@@ -0,0 +1,173 @@
+.. _efr32_radio:
+
+EFR32 Radio Boards
```

Comment:
Could we specifiy this a little bit more? This seems too generic for me.
If Silabs offers a new efr32 base board which provides a different pinout, we would have to add another erf32 radio board with a different name.
Could we include the base board name here (and also in the folder name)?

Reply:
> The name of the base board where the radio board can be plugged in is BRD4001A, the name of a typical radio board is BRD4104A. To avoid board names such as efr32_brd4001a_brd4104a, which look confusing, I decided to use the efr32_radio prefix for the boards. This resulted in names such as efr32_radio_brd4104a, efr32_radio_brd4250b. Comments on the naming scheme are welcome.

As I explained in my comment that's something I hoped to avoid. Especially for the board name, I don't mind the title. I mean, we can have something else than `efr32_radio_` for prefix but `efr32_brd4001a_` will be tedious to use in practice.

With boards (like with samples) we don't need to keep backwards compatibility so it's not a problem to rename the boards later, if SiLabs releases a new base board.